backends/sycl-ref/ceed-sycl-ref-qfunction-load.sycl.cpp

```diff
@@ -26,7 +26,7 @@
 //
 // TODO: Refactor
 //------------------------------------------------------------------------------
-extern "C" int CeedQFunctionBuildKernel_Sycl(CeedQFunction qf) {
+extern "C" int CeedSyclBuildQFunction(CeedQFunction qf) {
```

Comment:
```suggestion
extern "C" int CeedQFunctionBuildKernel_Sycl(CeedQFunction qf) {
```
Same here, keep original name.